src/components/player/cue-bar.tsx

```diff
@@ -129,8 +133,17 @@ const NoteCue: React.FC<any> = ({
 
   React.useEffect(() => {
     const isVisible = !muteNotes && cueActive && !seeking && playerReadyEnough
-    setVisible(isVisible)
-  }, [setPlayerPrefs, cueActive, seeking, muteNotes, playerReadyEnough])
+    if (!clickedOpen) {
```

Comment:
only run this if the cue wasn't clicked open, otherwise the logic stands. this prevents the "muted notes NEVER display even when I click them" issue @lsminter reported